pkg/cmd/app/app.go

```diff
@@ -0,0 +1,96 @@
+package app
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+
+	stream "github.com/GetStream/stream-chat-go/v5"
+	"github.com/GetStream/stream-cli/pkg/config"
+	"github.com/MakeNowJust/heredoc"
+	"github.com/spf13/cobra"
+)
+
+func NewCmds() []*cobra.Command {
+	return []*cobra.Command{getCmd(), updateCmd()}
+}
+
+func getCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use: "get-app --output-format [json]",
+		RunE: func(cmd *cobra.Command, args []string) error {
+			c, err := config.GetConfig(cmd).GetStreamClient(cmd)
+			if err != nil {
+				return err
+			}
+
+			r, err := c.GetAppConfig(cmd.Context())
+			if err != nil {
+				return err
+			}
+
+			format, _ := cmd.Flags().GetString("output-format")
+
+			if format == "json" {
+				unindented, err := json.Marshal(r.App)
+				if err != nil {
+					return err
+				}
+
+				indented := bytes.Buffer{}
+				err = json.Indent(&indented, unindented, "", "  ")
+				if err != nil {
+					return err
+				}
+
+				cmd.Println(indented.String())
+			} else {
+				return fmt.Errorf("unknown output format: %s", format)
```

Comment:
For better readability I would suggest to put this before and leave happy path out of `if` statement:
```go
if format != "json" {
	return fmt.Errorf("unknown output format: %s", format)
}

unindented, err := json.Marshal(r.App)
if err != nil {
	return err
}

indented := bytes.Buffer{}
err = json.Indent(&indented, unindented, "", "  ")
if err != nil {
	return err
}

cmd.Println(indented.String())

return nil
```

Feel free to ignore me :)

Reply:
Well this will probably be non relevant in coming PRs when more option will be added.

Reply:
yeah exactly, I kinda planned it for the next PRs. and I'll probably extract these to other places

Reply:
ignore me then